reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -375,6 +375,20 @@
 		return onAssembly(new FluxConcatIterable<>(sources));
 	}
 
+	/**
+	 * Concatenate all values provided as a vararg.
+	 *
+	 * @param values The values to concatenate
+	 * @param <T> The type of values in both source and output sequences
+	 *
+	 * @return a new {@link Flux} concatenating all source sequences
+	 * @since 3.1.2
+	 */
+	@SafeVarargs
+	public static <T> Flux<T> concat(T... values) {
```

Comment:
I don't get why, that's what `just(T...)` is for.

Reply:
what @akarnokd said, @yilinwei note the `concat` isn't necessary and then note the implementation becomes exactly the same as `just(T...)`:

```
    @SafeVarargs
	public static <T> Flux<T> just(T... data) {
		return fromArray(data);
	}
```

Reply:
@akarnokd @simonbasle That's because I'm an idiot attempting to commit at 1.30am...